devtools/ci/after_sucess.sh

```diff
@@ -2,20 +2,25 @@ coveralls
 
 echo $TRAVIS_PULL_REQUEST $TRAVIS_BRANCH
 
-if [[ "$TRAVIS_PULL_REQUEST" == "true" ]]; then
-    echo "This is a pull request. No deployment will be done."; exit 0
-fi
+#if [[ "$TRAVIS_PULL_REQUEST" == "true" ]]; then
+#    echo "This is a pull request. No deployment will be done."; exit 0
+#fi
+
+
+#if [[ "$TRAVIS_BRANCH" != "master" ]]; then
+#    echo "No deployment on BRANCH='$TRAVIS_BRANCH'"; exit 0
+#fi
 
+echo $BINSTAR_TOKEN
 
-if [[ "$TRAVIS_BRANCH" != "master" ]]; then
-    echo "No deployment on BRANCH='$TRAVIS_BRANCH'"; exit 0
+if [[ "2.7" =~ "$python" ]]; then
+    conda install --yes binstar jinja2
+    binstar -t ${BINSTAR_TOKEN}  upload  --force --u omnia -p openpathsampling-dev $HOME/miniconda/conda-bld/linux-64/opentis-dev-*
```

Comment:
This throws an error from banister saying `omnia` is not a valid directive, should be something like `upload`, etc... I assume that the token is not working and messes up the command somehow. Anyone seen this problem before?